configs/taco/MA.yaml

```diff
@@ -12,6 +12,17 @@ links:
   message: waiting a minute to load MA
 
 - name: secondary
+  url: https://www.mass.gov/info-details/covid-19-response-reporting
+  overseerScript: |
+    page.manualWait();
+    await page.waitForDelay(6000);
+    await page.waitForSelector("#tabZoneId854");
```

Comment:
looks good and good to merge, just a heads up that I've seen these kinds of IDs be unstable before so it may be worth keeping an eye on this one for a few days

Reply:
Thank you! I played around with this for a long time today. On the tableau version of this dash, the IDs changed with every refresh, but seemed stable on the MA page. I'm definitely going to keep an eye out!